doc/classes/KinematicBody2D.xml

```diff
@@ -80,6 +86,12 @@
 			<argument index="4" name="floor_max_angle" type="float" default="0.785398">
 			</argument>
 			<description>
+				Moves the body along a vector. [code]linear_velocity[/code] is a value in pixels per second. If the body collides with another, it will slide along the other body rather than stop immediately. If the floor the node stands on is also a KinematicBody2D, the node will follow the floor's motion. This works regardless of the method used to move the floor. You can use this to make moving or rotating platforms.
+				[code]floor_normal[/code] is the up direction, used to determine what is a wall and what is a floor or a ceiling. If set to the default value of [code]Vector2(0, 0)[/code], everything is considered a wall. This is useful for topdown games.
+				If the body is standing on a slope and the horizontal speed (relative to the floor's speed) goes below [code]slope_stop_min_velocity[/code], the body will stop completely. This prevents the body from sliding down slopes.
+				If the body collides, it will change direction a maximum of [code]max_bounces[/code] times before it stops.
+				[code]floor_max_angle[/code] is the maximum angle (in radians) where a slope is still considered a floor (or a ceiling), rather than a wall.
+				Returns the movement that remained when the body stopped. To get more detailed information about collisions that occured, use [method get_slide_collision].
```

Comment:
This is a complex concept, so a code example might be very useful here. A couple of suggestions:

- You mention the vel must be in "pixels per second". I think it should be more explicit that unlike in other places (like `move_and_collide`) you *don't* multiply by `delta` when using this method.